config/release.js

```diff
@@ -0,0 +1,18 @@
+/* jshint node:true */
+/* var RSVP = require('rsvp'); */
+var execSync = require('child_process').execSync;
+
+module.exports = {
+  publish: true,
+  afterPublish: function(project, versions) {
+    // Publish dummy app with docs to gh-pages
+    runCommand('ember github-pages:commit --message "Released ' + versions.next + '"');
```

Comment:
these are likely async, i suspect you actually want them to run in series not concurrently.